osama-breakout-room/Shirin Muataz/Style.css

```diff
@@ -0,0 +1,148 @@
+* {
+  padding: 0px;
+  margin: 0px;
+  box-sizing: border-box;
+}
```

Comment:
Great job on normalizing your page! 👌 This makes sure your page is consistent and shown properly across all browsers.